timesketch/lib/sigma_util.py

```diff
@@ -29,7 +29,7 @@
 logger = logging.getLogger('timesketch.lib.sigma')
 
 
-def get_sigma_config_file():
+def get_sigma_config_file(config_file=None):
```

Comment:
are these changes reflected in the `test_tools/timesketch/lib/sigma_util.py` file as well?

Reply:
There is no sigma_util.py in the test_tools, my idea was to use the same, that is why this https://github.com/google/timesketch/pull/1502/files#r536000064 is from the root timesketch folder instead of the ```test_tools``` folder.